### PR TITLE
[network path] Add network path configs

### PR DESCRIPTION
**Pull request by AlexandreYang**
### What does this PR do?

- Add config to config_template.yaml
- Add network_path.d/conf.yaml.example

### Motivation

Provide documentation of the options available for network path.

### Describe how to test/QA your changes

- Check new configs are present in datadog.yaml.example
- Check new configs are present in network_path.d/conf.yaml.example

### CODE REVIEW

File: pkg/config/config_template.yaml [hunk 1]
```diff
@@ -1637,6 +1637,34 @@ api_key:
   # enabled: false
 
 {{ end }}
+
+{{- if .NetworkPath }}
+{{- if (ne .OS "darwin")}}
```
Comment (AlexandreYang) [inline pkg/config/config_template.yaml:1642]: note for reviewers: we don't support macOS yet

**Review by hush-hush — APPROVED** [review]

**Review by ken-schneider — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

File: cmd/agent/dist/conf.d/network_path.d/conf.yaml.example [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ## Specifies the maximum number of hops (max time-to-live value) traceroute will probe.
+    #
+    # max_ttl: <PORT>
+
+    ## @param min_collection_interval - number - optional - default: 60
+    ## Specifies how frequently we should probe the endpoint.
+    ## Min collection interval is defined in seconds.
+    #
+    # min_collection_interval: 60
+
```
Comment (AlexandreYang) [inline cmd/agent/dist/conf.d/network_path.d/conf.yaml.example:38]: add source_service and destination_service here

File: cmd/agent/dist/conf.d/network_path.d/conf.yaml.example [hunk 3]
```diff
@@ -0,0 +1,46 @@
+init_config:
+    ## @param min_collection_interval - number - optional - default: 60
+    ## Specifies how frequently we should probe the endpoint.
+    ## Min collection interval is defined in seconds.
+    #
+    # min_collection_interval: 60
```
Comment (AlexandreYang) [inline cmd/agent/dist/conf.d/network_path.d/conf.yaml.example:6]: add `source_service` here

**Pull request merged**